backend/tests/test_auth_api.py

```
@@ -25,3 +27,135 @@
+def test_delete_product_not_found(client):
+    response = client.delete("/products/9999")
+    assert response.status_code == 404
+    assert response.json()["detail"] == "Product not found"
+
+
+def test_get_all_orders(client):
+    response = client.get("/orders")
+    assert response.status_code == 200
+    assert isinstance(response.json(), list)
+
+def test_update_order(client, db_session):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_update_order`

Add return type annotation: `None`

(ANN201)